src/lib/components/Toast.svelte

```diff
@@ -140,8 +140,62 @@
     padding: var(--padding-1_5x);
     box-sizing: border-box;
 
+    background: var(--toast-background, var(--overlay-background));
+    color: var(--toast-background-contrast, var(--overlay-background-contrast));
+
+    &.success {
+      --toast-background: var(--toast-background-success);
+      --toast-background-contrast: var(--toast-background-contrast-success);
+    }
+
+    &.info {
+      --toast-background: var(--toast-background-info);
```

Comment:
For consistency, should we invert the "background" and "info" because we have existing variables which are named such as `--toast-info-z-index` or `--toast-error-z-index`?